workflow.txt

```diff
@@ -1,10 +1,12 @@
-git clone
+fork on github
 git checkout -b my-solution
-git commit -m ""
-git fetch --all
+git clone
+  /* do your changes*/
 git add .
-git pull upstream master
-git merge my-solution
-fork on github
-git checkout master
-git push origin my-solution
+git commit -m ""
+git pull request origin master
```

Comment:
You need to push up your code before you can make a pull request. Also, you would be making the pull request on Github, not via the Terminal.